### PR TITLE
REMOVE FALSE POSITIVE TRIGGERS

### PR DESCRIPTION
**Pull request by RayDeeUx**
edit since i cant edit PR title: REMOVE FALSE POSITIVE CRITERIA

### CODE REVIEW

File: files/botautoresponse.json [hunk 1]
```diff
@@ -681,7 +678,6 @@
         "triggers": [
             [
                 ".forge",
```
Comment (MicrocontrollersDev) [inline files/botautoresponse.json]:
```suggestion
                ".forge"
```

Comment (RayDeeUx) [reply files/botautoresponse.json:680]: AAAAAAAA

Comment (KTibow) [reply files/botautoresponse.json]: a:+1:

**Comment by MicrocontrollersDev** [comment]
PLEASE STOP YELLING OR ELSE I WILL REPORT YOU TO MOJANG AND GET YOU PERMANENTLY BANNED FROM MULTIPLAYER

**Comment by Mqisty** [comment]
PLEASE DO NOT YELL MICROCONTROLLERSDEV

**Pull request closed without merging**